app/src/main/java/fr/free/nrw/commons/category/CategoryDetailsActivity.java

```diff
@@ -120,6 +120,7 @@ public void onItemClick(AdapterView<?> adapterView, View view, int i, long l) {
         if (mediaDetails == null || !mediaDetails.isVisible()) {
             // set isFeaturedImage true for featured images, to include author field on media detail
             mediaDetails = new MediaDetailPagerFragment(false, true);
+            //FIXME: Is this even required. Check it once.
```

Comment:
Which line are you referring to here? `supportFragmentManager` is used twice below.

Reply:
This wasn't supposed to be here. It was just a point for me to check. Will remove.